webpack.config.js

```diff
@@ -39,7 +39,7 @@ const sharedConfig = {
       new TerserPlugin({
         parallel: true,
         sourceMap: false,
-        cache: true,
+        cache: false,
```

Comment:
If there's issues with caching we should track that separately. Perhaps there are alternative solutions.

Reply:
@wassgha can you please clarify why did you apply this change? Can I revert it or it is a requirement for this PR works properly?

Reply:
Ditto, resolved.